pkg/utils/random.go

```diff
@@ -34,3 +34,21 @@ func RandomString(n int) string {
 func RandomJitter(duration, maxJitter time.Duration) time.Duration {
 	return (duration + time.Duration(rand.Float64()*float64(maxJitter))).Truncate(time.Millisecond) //nolint:gosec // don't need crypto/rand
 }
+
+// Truncate string.
+func TruncateString(str string, length int) string {
+	if length <= 0 {
+		return ""
+	}
+
+	truncated := ""
+	count := 0
+	for _, char := range str {
+		truncated += string(char)
+		count++
+		if count >= length {
+			break
+		}
+	}
+	return truncated
+}
```

Comment:
I don't think random package is the best place for this function. Maybe this function can be used instead https://github.com/kubernetes/utils/blob/d93618cff8a2/strings/strings.go#L41